src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -109,6 +111,34 @@ func runClientProxy(cfg *config.Config) {
 	// Configure client initially
 	client.Configure(cfg.Client)
 
+	// this flag specifies the port to open the HTTP server
+	// between the UI and
+	// with a corresponding HTTP server at
+	// the following address
+	if cfg.Client.UiAddr != "" {
+		wlChan := make(chan *whitelist.Config)
+		go func() {
+			http.ListenAndServe(cfg.Client, configUpdates, wlChan)
```

Comment:
I read that this is not `ListenAndServe` from the `net/http` package, but from the `github.com/getlantern/flashlight/http` and it does not return an error value, so there's nothing to catch here actually. I think it would be useful to return an error value on this situation, so we could print it and exit the application.